wiki/usage.md

```diff
@@ -23,12 +23,18 @@ $ docker build --tag bats:latest .
 $ docker run -it bats:latest --tap /opt/bats/test
 ```
 
-To mount your tests into the container, first built the image as above. Then:
+To mount your tests into the container, first built the image as above. Then, for example with `bats`:
 ```bash
-$ docker run -v $(pwd)/test:/test bats /test
+$ docker run -it -v $(pwd):/opt/bats bats:latest /opt/bats/test
 ```
 This has run the `test/` directory from the bats-core repository inside the bats Docker container.
 
+
+Or for test files without any dependencies:
```

Comment:
I think I understand now what you're saying... It would help to explain some more, something like: "For test suites that are intended to run in isolation from the project (i.e. no tests depend on project files outside of the test directory), you can mount the test directory in isolation and execute them like so:"